package.json

```diff
@@ -21,12 +21,13 @@
     "watch": "node tasks/watch.js",
     "test": "mocha src/**/*spec.js",
     "test:ci": "nyc --reporter=lcov mocha",
-    "stylelint": "stylelint src/scss/**/*.scss src/scss/*.scss --color --formatter verbose",
-    "eslint": "eslint src/js",
-    "phplint": "vendor/squizlabs/php_codesniffer/bin/phpcs --standard=sparkpress-standard.xml --extensions=php ./src/php",
-    "node-version": "check-node-version --package",
-    "twiglint": "twigcs src/php/views",
-    "lint": "run-p phplint stylelint eslint twiglint"
+    "lint:js": "eslint src/js",
+    "lint:php": "./scripts/run.sh vendor/bin/phpcs --standard=sparkpress-standard.xml --extensions=php src/php",
+    "lint:scss": "stylelint src/scss/**/*.scss src/scss/*.scss --color --formatter verbose",
+    "lint:twig": "./scripts/run.sh vendor/bin/twigcs src/php/views",
+    "lint": "run-p lint:*",
```

Comment:
I think `run-s` might be a little better for this type of script. That will keep the results separated instead of possibly jumbling SCSS and PHP errors together, and then when one of the scripts exits with a non-zero code, the whole things will stop and you can focus on fixing those errors.

Reply:
Good call! Updated.